packages/api/es/indexer.js

```diff
@@ -56,7 +56,7 @@ async function indexLog(esClient, payloads, index = defaultIndexAlias, type = 'l
         record = JSON.parse(p.message);
       }
       // level is number in elasticsearch
-      if (typeof record.level === 'string') record.level = log.convertLogLevel(record.level);
```

Comment:
This doesn't catch all of the cases that could be a string.  Lodash's `isString` method contains these additional checks:

```
function isString(value) {
  return typeof value == 'string' ||
    (!isArray(value) && isObjectLike(value) && baseGetTag(value) == stringTag);
}
```